setup.py

```diff
@@ -48,11 +48,18 @@ def parse_requirements(filename):
         ]
     },
 
-    url=r'http://cmake.org/',
-    download_url=r'https://cmake.org/download',
+    url='https://cmake.org/',
```

Comment:
This is an `https` url, and also, `/` is valid in a regular string, so removed the `r`.